examples/huddly_l1/detections.js

```diff
@@ -0,0 +1,30 @@
+const HuddlyDeviceApiIp = require('@huddly/device-api-ip').default;
+const HuddlySdk = require('@huddly/sdk').default;
+const { HUDDLY_L1_PID } = require('@huddly/sdk/lib/src/components/device/factory').default;
+
+const ipApi = new HuddlyDeviceApiIp();
+const sdk = new HuddlySdk(ipApi);
+let manager;
+let detector;
+
+sdk.on('ATTACH', async (aceDevice) => {
+  if (aceDevice.pid == HUDDLY_L1_PID) {
+    manager = aceDevice;
+    detector = manager.getDetector({ objectFilter: ['person'], DOWS: true, rawDetections: true });
+    detector.on('DETECTIONS', (data = []) => {
```

Comment:
I think the data = [] as default is misleading, as it should never really happen given proper implementation on the sdk.